skyreader/result.py

```diff
@@ -946,10 +974,27 @@ def bounding_box(ra, dec, theta, phi):
             contour_colors=['k', 'r', 'g', 'b'][:3]
 
         sample_points = np.array([np.pi/2 - grid_dec, grid_ra]).T
+        
         # Call meander module to find contours
-        contours_by_level = meander.spherical_contours(sample_points,
-            grid_value, contour_levels
-            )
+        if not circular:
+            contours_by_level = meander.spherical_contours(sample_points,
+                grid_value, contour_levels
+                )
+        if circular:
+            sigma50 = np.deg2rad(circular_err50)
+            sigma90 = np.deg2rad(circular_err90)
+            Theta50, Phi50 = self.circular_contour(minRA, minDec, sigma50, nside)
+            Theta90, Phi90 = self.circular_contour(minRA, minDec, sigma90, nside)
+            contour50 = list()
+            for i in range(len(Theta50)):
+                contour50.append([Theta50[i], Phi50[i]])
+            contour50 = np.asarray(contour50)
+            contour90 = list()
+            for i in range(len(Theta90)):
+                contour90.append([Theta90[i], Phi90[i]])
+            contour90 = np.asarray(contour90)
+            contours_by_level = [[contour50], [contour90]]
```

Comment:
Can we build the `contour` arrays from `Theta` and Phi` using just numpy stuff instead of building intermediate lists?

Reply:
I did build the contours specular as the millipede ones. Also these ones are lists. I tried using np.append(), but then I couldn't keep the lists of two elements for the single points. If you think it is necessary I can further modify the building of the contours.

Reply:
I guess the existing code is not the best style-wise :)

The final shape looks like it's `(n, 2)` and you start from two arrays of shape `(n,)`. If I recall correctly, this could be done with `np.vstack()`.